cmd/src/users_clean.go

```diff
@@ -0,0 +1,202 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/sourcegraph/src-cli/internal/api"
+)
+
+func init() {
+	usage := `
+Examples:
+
+	$ src users clean -d 182
+
+`
+
+	flagSet := flag.NewFlagSet("clean", flag.ExitOnError)
+	usageFunc := func() {
+		fmt.Fprintf(flag.CommandLine.Output(), "Usage of 'src users %s':\n", flagSet.Name())
+		flagSet.PrintDefaults()
+		fmt.Println(usage)
+	}
+	var (
+		daysToDelete = flagSet.Int("d", 365, "Day threshold on which to remove users, defaults to 365")
+		noAdmin      = flagSet.Bool("no-admin", false, "Omit admin accounts from cleanup")
+		// TODO: Add an email functionality to email removed users, open editor like in git commit
+		// toEmail            = flagSet.Bool("email", false, "send removed users an email")
+		removeNoLastActive = flagSet.Bool("removeNeverActive", false, "removes users with null lastActive value")
+		skipConfirmation   = flagSet.Bool("skip-conf", false, "skips user confirmation step allowing programmatic use")
+		// TODO: Write json file containing users to be deleted, last active usage, site-admin status, and emails
+		// json               = flagSet.Bool("json", false, "Write json file containing users to be deleted, last active usage, site-admin status, and emails")
+		apiFlags = api.NewFlags(flagSet)
+	)
+
+	handler := func(args []string) error {
+		if err := flagSet.Parse(args); err != nil {
+			fmt.Printf("failed to parse command args with error: %s", err)
+		}
+
+		ctx := context.Background()
+		client := cfg.apiClient(apiFlags, flagSet.Output())
+
+		vars := map[string]interface{}{
+			"-d": api.NullInt(*daysToDelete),
```

Comment:
That's not used, right?

Reply:
right 😓 - and now I better understand how the `api` package works 😁